spot-ml/ml_ops.sh

```diff
@@ -66,6 +66,8 @@ LDA_OUTPUT_DIR=${DSOURCE}/${FDATE}
 
 TOPIC_COUNT=20
 
+LDA_IMP="LDAC"
```

Comment:
I have discussed this with Nate Smith recently and this is pretty much what he is doing in his new conf file and ml_ops that he is writing